bee/src/main/java/com/apisense/bee/ui/fragment/HomeFragment.java

```diff
@@ -75,6 +93,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onResume() {
         super.onResume();
         retrieveActiveExperiments();
+        autoUpdateRunning = new Timer();
+        autoUpdateRunning.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                getActivity().runOnUiThread(new Runnable() {
+                    public void run() {
+                        retrieveActiveExperiments();
+                    }
+                });
+            }
+        }, 0, 1000); // updates each second
```

Comment:
With an update interval of 1000 ms, a click not responding issue appears. It could be nice if we put a value of `3000` instead of `1000`.